data/map planets.txt

```diff
@@ -277,6 +286,15 @@ planet Baianus
 	description `	Unlike Titan, however, Baianus is regularly wracked by both waves of heat and energy, and a stay of any length on the surface is likely to see waves of auroras accompanied by glows and sparks rippling across the terrain. Your environmental system strongly recommends remaining inside your ship whenever ion storms are active.`
 	government Uninhabited
 
+planet Bailey
+	attributes ccor mining
+	landscape land/lava3
+	description `Bailey is a dead, nearly tidally locked world scarcely larger and more habitable than Mercury. Its cool side has hardened into a mineral-rich crust, a rarity in this region of space. Despite its inhospitable conditions, Bailey's bountiful crust has brought several mining expeditions to the planet.`
+	description `	Like aphids sucking the sap from a plant, semi-mobile mining stations strip and refine the minerals, a method originally developed by the Confederated Councils. Since their fall, the pirates of the region have maintained the stations - although with considerably less organization than what was upheld by the central government.`
+	spaceport `Given the roving nature of the settlements, and the lack of a livable atmosphere, the spaceport and other major infrastructure are located in orbit. Transit to and from the planet is facilitated by a clever network of skyhooks, which utilize fewer amounts of rare and expensive resources compared to the more common repulsor technology used elsewhere.`
```

Comment:
```suggestion
	spaceport `Given the roving nature of the settlements, and the lack of a livable atmosphere, the spaceport and other major infrastructure are located in on a repulsor-lift facility hovering just above a few feet above Bailey's surface. Floating in place thanks to the lack of atmosphere the planet slowly rotates beneath it while the facility is always on the dark side of the planet.`
```

Reply:
We use metric in Endless Sky, so instead of `a few feet` put `a meter` instead.

Reply:
A meter is not enough. Even minor topographical features (mountains???) would ram into the spaceport as it glides along the surface.

Reply:
> A meter is not enough. Even minor topographical features (mountains???) would ram into the spaceport as it glides along the surface.

Bailey is a semi-molten lava world. The crust melts during the day and solidifies at night. To note: as Bailey is only just barely not tidally locked the day/night cycle is close to a local year. Features would be minimal here.